src/swagger-specs/query-service.yaml

```diff
@@ -776,6 +779,311 @@ paths:
+        description: Update the status of an alert to either `enable` or `disable` by providing a query or schedule ID and an alert type.
+        consumes:
+          - application/json
+        produces:
+          - application/json
+        parameters:
+          - $ref: '#/parameters/authorization'
+          - $ref: '#/parameters/x-gw-ims-org-id'
+          - $ref: '#/parameters/x-api-key'
+          - $ref: '#/parameters/x-sandbox-name'
+          - $ref: '#/parameters/x-request-id'
+          - $ref: '#/parameters/User-Agent'
+
+          - name: assetId
+            description: A query ID or a schedule ID.
+            required: true
+            in: path
+            type: string
+
+          - name: alertType
+            description: This property determines the state of query execution that triggers a user to receive an alert. The alert type is one of three values (start, success, failure).
+            required: true
+            in: path
+            type: string
+
+          - name: operation
+            in: body
+            description: The requested operation.
+            required: true
+            schema:
+              type: object
+              properties:
+                op:
+                  type: string
+                  description: The operation to be performed. The only accepted value is 'replace'. It replaces the current alerts' status with the value provided in the request.
+                  enum:
+                    - replace
+                path:
+                  type: string
+                  description: The path must be /status to ensure the alert status is updated.
+                  enum:
+                    - /status
+                status:
+                  type: string
+                  description: The provided value will be the status of the alert after the update takes effect.
+                  enum:
+                    - enable
+                    - disable
+
+        responses:
+          200:
+            description: "The alert status has been updated successfully."
+            schema:
+              $ref: '#/definitions/alert_patched'
+          400:
+            description: "Bad Request. The payload may be malformed, or an invalid alertType may have been provided in URL."
+            schema:
+              $ref: '#/definitions/alert_patched_bad_request_error_message'
+          401:
+            description: "Unauthorized"
+          404:
+            description: "No alert found for the assetId and alertType provided."
+            schema:
+              $ref: '#/definitions/alert_patched_not_found_error_message'
+          500:
+            description: "Internal Server Error"
+    delete:
+      operationId: "alertDelete"
+      tags:
+        - "Alert subscriptions"
+      summary: Delete the alert for a particular query and alert type
+      description: This request deletes the alert for a particular query or schedule ID and alert type.
```

Comment:
A bit confused how this is phrased. Is the dichotomy:

- (Query OR Schedule ID) AND Alert Type

or:

- Query OR (Schedule ID AND Alert Type)

Please rephrase so this is clearer.